armbian/base/scripts/set-bitcoin-network.sh

```diff
@@ -5,7 +5,7 @@ set -e
 # 
 
 function usage() {
-  echo "BitBox Base: set Bitcoin network"
+  echo "BitBox Base: set Bitcoin network (DEPRECATED)"
```

Comment:
Do we really need to deprecate this script? Couldn't we just remove it, to avoid the overhead of maintaining two scripts for some time?

Reply:
Definitely, just wanted to test it first. Removed.